bin/make/frameworks.sh

```diff
@@ -1,7 +1,58 @@
 #!/usr/bin/env bash
 
 rm -rf build
-xcodebuild -target FBSimulatorControl
-xcodebuild -target FBDeviceControl
-xcodebuild -target FBControlCore
-xcodebuild -target XCTestBootstrap
+
+set -e
+set -o pipefail
+
+if [ "${XCPRETTY}" = "0" ]; then
+  USE_XCPRETTY=
+else
+  USE_XCPRETTY=`which xcpretty | tr -d '\n'`
+fi
+
+if [ ! -z ${USE_XCPRETTY} ]; then
+  XC_PIPE='xcpretty -c'
+else
+  XC_PIPE='cat'
+fi
+
+BUILD_DIR="build"
+XC_PROJECT="FBSimulatorControl.xcodeproj"
+
+xcrun xcodebuild \
+  -SYMROOT="${BUILD_DIR}" \
```

Comment:
@jmoody I probably would have just made this a function but I'm not going to block for a nitpick